packages/flutter_tools/lib/src/macos/xcode.dart

```diff
@@ -18,11 +18,11 @@ import '../base/version.dart';
 import '../build_info.dart';
 import '../ios/xcodeproj.dart';
 
-Version get xcodeRequiredVersion => Version(12, 0, 1, text: '12.0.1');
+Version get xcodeRequiredVersion => Version(12, 3, null, text: '12.3');
 
 /// Diverging this number from the minimum required version will provide a doctor
 /// warning, not error, that users should upgrade Xcode.
-Version get xcodeRecommendedVersion => Version(13, 0, 0, text: '13.0.0');
+Version get xcodeRecommendedVersion => Version(13, null, null, text: '13');
```

Comment:
Just realized this should have been `13, null, null` so it gets printed as `Xcode 13` and not `Xcode 13.0.0`.